brain/python/compartmentReportReader.cpp

```diff
@@ -105,46 +99,29 @@ CompartmentReportMappingProxy CompartmentReportView_getMapping(
     return view;
 }
 
-CompartmentReportFramePtr CompartmentReportView_loadAt(
-    CompartmentReportView& view, float time)
+
+bp::object CompartmentReportView_loadAt(CompartmentReportView& view, double time)
 {
-    return CompartmentReportFramePtr(
-        new CompartmentReportFrame(view.load(time).get()));
+    return frameToTuple(view.load(time).get());
```

Comment:
I've noticed that the C++ implementation of this function has a bug. If passes a timestamp of the frame the requested timestamp when it should return the timestamp of the start of the time window of the returned frame. 
Furthermore, the frame could be empty, which happens when the requested timestamp is outside the report boundaries. In this case you have to check frame.empty() and return None if that's the case.